src/components/farmers/FarmerCard.css

```diff
@@ -0,0 +1,15 @@
+ion-card.farmer-card {
```

Comment:
can we not do inline styles w ionic? Something similar to tailwindcss?

Reply:
I'd love to use tailwind, but we already use ionic components so plain CSS will be enough for customization.

Yeah, we can inline style still works. It's a matter of preference.

Reply:
Agreed. Feel free to do whatever you feel faster with